src/test/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticatorTest.java

```diff
@@ -127,7 +128,7 @@ public void basicTest() throws Exception {
             .put(IDP_METADATA_URL, mockSamlIdpServer.getMetadataUri())
             .put("kibana_url", "http://wherever")
             .put("idp.entity_id", mockSamlIdpServer.getIdpEntityId())
-            .put("exchange_key", "abc")
+            .put("exchange_key", "secret_key_length_should_be_at_least_256_bits_or_86_characters_in_lengthabcdefghabcdef")
```

Comment:
This is a strange requirement, I'm guessing that CXF RS was padding the key - can you see if we can automatically pad the keys in the same was as was being done with CXF, otherwise we wouldn't be able to migrate folks that are already using 'undersized' keys?